openshift/unleash.yaml

```diff
@@ -25,6 +25,18 @@ objects:
         labels:
           app: ${identifier}
       spec:
+      affinity:
```

Comment:
indent this section 2 to the right?

deployment started failing after this merge: https://console-openshift-console.apps.appsrep05ue1.zqxk.p1.openshiftapps.com/k8s/ns/app-sre-pipelines/tekton.dev~v1beta1~PipelineRun/app-interface-unleash-app-sre-stage-202205042107/logs/openshift-saas-deploy